x-pack/plugins/apm/typings/es_schemas/Span.ts

```diff
@@ -39,6 +39,15 @@ export interface Span extends APMDoc {
     subtype: string;
     sync: boolean;
     type: string;
+    http?: {
```

Comment:
Do you want to add the other values for `span.http` here?
https://github.com/elastic/apm-server/blob/master/_meta/ecs-migration.yml#L198-L211